src/shared/sam/debugger/commands/addSamDebugConfiguration.ts

```diff
@@ -128,10 +128,15 @@ export async function addSamDebugConfiguration(
             return
         }
     } else if (type === API_TARGET_TYPE) {
-        const preloadedConfig = {
-            path: apiEvent?.Properties?.Path,
-            httpMethod: apiEvent?.Properties?.Method,
-            payload: apiEvent?.Properties?.Payload,
+        let preloadedConfig = undefined
+        if (apiEvents && apiEvents.size === 1) {
+            for (const event of apiEvents.values()) {
+                preloadedConfig = {
```

Comment:
strange to use a for-loop here, since it is reassigning `preloadedConfig`. If we want to guard against accidentally forgetting about this area when we later support `apiEvents.size > 1`, let's throw an error.